nutanix/resource_nutanix_ndb_register_database.go

```diff
@@ -0,0 +1,697 @@
+package nutanix
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	era "github.com/terraform-providers/terraform-provider-nutanix/client/era"
+	"github.com/terraform-providers/terraform-provider-nutanix/utils"
+)
+
```

Comment:
I see computed values similar to that of create database resource, scaling, restore, etc. computed values. Here we can do this 
```suggestion
func resourceNutanixNDBRegisterDatabase() *schema.Resource {

        schema = &schema.Resource{
		CreateContext: resourceNutanixNDBRegisterDatabaseCreate,
		ReadContext:   resourceNutanixNDBRegisterDatabaseRead, 
		.
		.
		.
        // add computed values to existing schema
	schema.Schema.update(getComputedDatabaseShchema())
	return schema
		                        
```

Reply:
".update()" is just for logical explanation.

Reply:
will make the changes later as improvement.